candi/cloud-providers/azure/terraform-modules/static-node/main.tf

```diff
@@ -25,6 +25,11 @@ data "azurerm_subnet" "kube" {
 locals {
   zones_count = length(local.zones)
   zone        = local.zones[var.nodeIndex % local.zones_count]
+  default_cloud_config = <<-EOF
+  #cloud-config
+  mounts:
+  - [ ephemeral0, /mnt/resource ]
+  EOF
```

Comment:
why we need this cloud_config?

Reply:
I have added this configuration for cloudPermanents nodes, similar to the configuration for master nodes. The same configuration for master nodes has been added in this commit: https://github.com/deckhouse/deckhouse/commit/2dafe7794f987593a8cd53cfd2dcc0126217f975.